docs/site/package.json

```diff
@@ -13,21 +13,21 @@
     "clean": "rimraf build"
   },
   "devDependencies": {
-    "babel-cli": "^6.18.0",
+    "babel-cli": "^6.22.2",
```

Comment:
> ^6.0.0, ^6.18.0 & ^6.22.2 are inherently the same as I understand it

What do you mean? Yarn is locking the dependencies. The scope is only affecting upgrades.